requirements/requirements.txt

```diff
@@ -0,0 +1,9 @@
+pytest
```

Comment:
Is there an advantage to having requirements.txt over the requirements that are designated in setup.py? My understanding was that they're equivalent.